docs/src/QuickStart.md

```diff
@@ -72,5 +69,16 @@ result = simp(x0)
 # Visualize the result using Makie.jl
 
 glmesh = GLMesh(problem, result.topology)
-mesh(glmesh)
+
+# workaround : https://github.com/JuliaPlots/Makie.jl/issues/647
+function GeometryBasics.Mesh(glmesh::GeometryTypes.GLNormalVertexcolorMesh)
```

Comment:
I would just make `GLMesh` do this by default then we don't need the extra step.